app/src/main/kotlin/com/wire/android/ui/userprofile/image/AvatarPicker.kt

```diff
@@ -58,11 +61,7 @@ fun AvatarPickerScreen(viewModel: AvatarPickerViewModel) {
             viewModel.navigateBack()
         },
         onSaveClick = {
-            if (state.avatarPickerFlow.pictureState is PictureState.Picked) {
-                viewModel.uploadNewPickedAvatarAndBack(state.avatarPickerFlow.pictureState.avatarUri, context)
-            } else {
-                viewModel.navigateBack()
-            }
+            viewModel.uploadNewPickedAvatarAndBack(viewModel.pictureState.avatarUri)
```

Comment:
I do not think we need to pass an argument into the function as it is already in the viewmodel